src/app/src/pages/Profile.tsx

```diff
@@ -152,6 +160,35 @@ export default function Profile({ currentUser }: Props): JSX.Element {
     fetchData();
   }, [author?.id]);
 
+  // Get Current User's followers and update isFollwing is the author is being followed by the current user
+  const [currentUserFollowers, setCurrentUserFollowers] = useState<Author[]>(
+    []
+  );
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const res = api.authors.withId("" + currentUser?.id).followings.list();
+        res.then((followers) => {
+          setCurrentUserFollowers(followers);
+        });
+        let isFollowing = false;
+        if (currentUserFollowers) {
+          for (let i = 0; i < currentUserFollowers.length; i++) {
```

Comment:
This can be simplified into something like:
```
        if (currentUserFollowers) {
          let obj = currentUserFollowers.find(author => author.id === `${author?.id}`);
          if(obj){
            isFollowing=true;
          }
        }
```

Reply:
using API call now, so this part is removed